spack/packages/dla-future-fortran/package.py

```diff
@@ -30,6 +30,7 @@ class DlaFutureFortran(CMakePackage):
 
     # Requires 0.3.0 for complex API
     # Requires 0.4.0 for Intel OneAPI MKL
+    # Requires 0.4.1 for CMake version
```

Comment:
Thanks, I'll remove this comment. It was meant initially to remind me allow `0.3.0` when MKL is not used, but since we are going to use the fixed CMake version it no longer matters.